webdriver/tests/support/fixtures.py

```diff
@@ -12,6 +13,20 @@
 default_host = "http://127.0.0.1"
 default_port = "4444"
 
+logger = mozlog.get_default_logger()
+
+
+def ignore_exceptions(f):
+    def inner(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except webdriver.error.WebDriverException as e:
+            logger.warning("Ignored exception %s" % e)
+    inner.__name__ = f.__name__
+    return inner
+
+
+@ignore_exceptions
```

Comment:
I’m OK with ignoring errors in here as the tests may also have caused the driver to end up in a bad state, and the tests should do the actual assertions that the the driver is compliant.